modules/reitit-frontend/src/reitit/frontend/history.cljs

```diff
@@ -68,6 +68,7 @@
               (let [uri (.parse Uri (.-href el))]
                 (when (and (or (and (not (.hasScheme uri)) (not (.hasDomain uri)))
                                (= current-domain (.getDomain uri)))
+                           (not (.closest el "[contenteditable]"))
```

Comment:
Afaik `closest` is not supported in IE without polyfill. `goog.dom.getAncestor` should be better choice for compatibility: 
`(dom/getAncestor el (fn [node] (.isContentEditable node)))`. And because this check is probably not the fastest, it is probably best to move it to end the `and` so that if `and` is short-circuited, this check doesn't have to run.

Reply:
Thanks! I just pushed your suggestions.